src/main/java/mekanism/common/registries/MekanismModules.java

```diff
@@ -89,6 +90,8 @@ private MekanismModules() {
                 .rendersHUD().exclusive());
     public static final ModuleRegistryObject<?> ELYTRA_UNIT = MODULES.registerMarker("elytra_unit", () -> MekanismItems.MODULE_ELYTRA.asItem(),
           builder -> builder.rarity(Rarity.EPIC));
+    public static final ModuleRegistryObject<ModuleGyroscopicStabilizationUnit> GYROSCOPIC_STABILIZATION_UNIT = MODULES.register("gyroscopic_stabilization_unit",
```

Comment:
Use `registerMarker` instead of using `register` and creating an empty dummy class implementation.

Reply:
I even looked for a way to do that, apparently not hard enough though.